src/plots/cartesian/layout_attributes.js

```diff
@@ -797,6 +797,7 @@ module.exports = {
         editType: 'ticks',
         description: 'Sets the width (in px) of the grid lines.'
     },
+    griddash: extendFlat({}, dash, {dflt: '', editType: 'ticks'}),
```

Comment:
There is no empty value for `dash` attribute. The `dflt` is *solid* by default.
```suggestion
    griddash: extendFlat({}, dash, {editType: 'ticks'}),
```